core/templates/pages/learner-dashboard-page/learner-dashboard-page.component.ts

```diff
@@ -232,9 +232,28 @@ angular.module('oppia').component('learnerDashboardPage', {
         return value;
       };
 
+      var changeDelay = function() {
+        if (ctrl.checkMobileView()) {
+          return 1000;
+        }
+        return 0;
```

Comment:
```suggestion
        return ctrl.checkMobileView() ? 1000 : 0;
```

Also, 1000 and 0 seems like magic numbers. Let's use a named constant instead that explains what these represent.

Reply:
DONE